explorer/src/explorer/Main.hs

```diff
@@ -48,8 +50,10 @@ printFlags = do
 main :: IO ()
 main = do
     args <- getExplorerOptions
-    printFlags
-    runProduction $ action args
+    let loggingParams = CLI.loggingParams "node" (enaCommonNodeArgs args)
+    loggerBracket loggingParams . handleAny (usingLoggerName "auxx" . logError . show) $ do
```

Comment:
`usingLoggerName "explorer"`?

Reply:
Well, I took the names straightly from #1753, but I think that is a good point.